packages/e2e-test-utils/src/inserter.js

```diff
@@ -99,13 +99,31 @@ async function waitForInserterCloseAndContentFocus() {
 	);
 }
 
+async function selectInserterTab( tabName ) {
+	try {
```

Comment:
TODO: I need to find a way to avoid this overhead of checking if the appropriate tab is selected.